c++/src/kj/common.h

```diff
@@ -1928,6 +1929,51 @@ class ArrayPtr: public DisallowConstCopyIfNotConst<T> {
     return true;
   }
 
+  inline auto operator<=>(decltype(nullptr)) const { 
```

Comment:
why is it auto? can't it be `std::strong_ordering`?

Reply:
if the internal type is double then it needs to be `partial_ordering`.

Reply:
There should now be comments that properly address this.